core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -565,7 +565,8 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     private PingResponse createPingResponse(DiscoveryNodes discoNodes) {
-        return new PingResponse(discoNodes.getLocalNode(), discoNodes.getMasterNode(), clusterName, contextProvider.nodeHasJoinedClusterOnce());
+        return new PingResponse(discoNodes.getLocalNode(), discoNodes.getMasterNode(), clusterName,
```

Comment:
So long as we do it one single way I'm fine with either way. Personally I prefer not having the prefix as well but I thought like Luca that we were standardizing one having it.